EngineeringModel/ViewModels/ElementDefinitionBrowser/ElementDefinitionTreeRows/ParameterOrOverrideBaseRowViewModel.cs

```diff
@@ -259,28 +259,48 @@ private ParameterValueSetBase GetValueSet()
         /// </param>
         private void SetProperties(ParameterValueSetBase valueSet)
         {
-            this.Value = valueSet.ActualValue.Any() ? valueSet.ActualValue.First() : "-";
+            this.Value = this.GetDisplayValueFromValueSet(valueSet.ActualValue);
 
             if (this.ContainedRows.Count == 0)
             {
                 this.ScaleShortName = this.Thing.Scale == null ? "-" : this.Thing.Scale.ShortName;
             }
 
             this.Switch = valueSet.ValueSwitch;
-            this.Formula = valueSet.Formula.Any() ? valueSet.Formula.First() : "-";
-            this.Computed = valueSet.Computed.Any() ? valueSet.Computed.First() : "-";
+            this.Formula = this.GetDisplayValueFromValueSet(valueSet.Formula);
```

Comment:
i don't understand why the logic for `Formual` and `Computed` is not the same as for `Manual` and `Reference`. Can you explain?

Reply:
@samatrhea: I don't know either, but as you can see it was already the case, so someone probably did this on purpose.
I did not want to question this, but if you do... Should we use the same logic for all properties and see what happens?